lib/new-relic-client.js

```diff
@@ -74,7 +93,7 @@ module.exports = CoreObject.extend({
     possible = possible.filter(function(path) {
       var tempFilePath = absoluteBaseName(path);
       var pathWithoutPrefix = tempFilePath.replace(fileName, '');
-      return !pathWithoutPrefix || /^-[a-zA-Z0-9]+$/.test(pathWithoutPrefix);
+      return !pathWithoutPrefix || /^[-a-zA-Z0-9]+$/.test(pathWithoutPrefix);
```

Comment:
Wondering if this is not going to break original case scenario?

Reply:
we need a working original scenario to test that, for our current scenario this is the only way we could make it work.

in our case `pathWithoutPrefix` is `"bv-barebone-a8e0254872aaad594220b9268ec6e78c"`
so the current regex does not work for that path.